### DISCUSSION TITLE
The 'Third' Reward

### DISCUSSION THREAD

**Discussion by VV-LX** [ppy/osu#28837] [Ideas]
### Why

In osu, there are only two rewards for playing a beatmap the game intentionally and that the developers control: XP and PP. The latter has a large influence on how players and mappers interact with the game. Though more significant in the past, this aspect of osu is still exerting negative effects on both communities.

I believe osu lacks a critical element that, if implemented, will be a net positive for the whole. This feature is osu!Coins, and is the core element that’ll create a new way to direct player engagement.

Right off the BAT, I’ll address concerns about osu quickly turning into a P2W/Cosmetics-over-Gameplay. 
osu!Coins will never be purchasable with real currency. They will be obtainable only through players engaging with content the developers want to ‘spotlight’. Beatmaps will not be locked behind purchase requirements.

### osu!Coins 
The third leg to XP and PP. Given when reaching Level Milestones, Clearing Quests/Challenges, Earning Medals, Daily Logins, etc. Used to reward any kind of Activity that needs to be incentivized.

### Usage
Where would osu!Coins be used? The Peppy Grand Casino of course! That’s a joke. 
osu!Coins should be spent on appearance-altering items (or Cosmetics, though that feels like a he-who-should-not-be-named around here). What sort of alterations? Well, my beloved developer, You are God, and we’re just lambs in your world. If the current realms of the webpage and game aren’t good enough, you could even make a whole new client (Oh right, lazer). Adding a new way to let players choose menu backgrounds doesn’t sound so daunting in comparison.

### Coin Shop 
Where Coins can be exchanged for Cosmetics, Official Event Entry Tickets, Perks, etc.

### Cosmetics 

Animated Icons, Chat Emotes, Friend List Expansion, Covers, and anything else that’s possible but isn’t available at present. Having custom versions of these obtainable during special events (such as possible collaborations with other IP’s) would be very enticing. Having them be community-made would be even better.  Having these pieces be encouraged and validated through a process like the Skin Contests and Background Art contests would be in line with osu's community-generated nature.

 osu’s in-the-works [Story Mode](https://github.com/ppy/osu/discussions/22270)    could also effectively draw inspiration/themes for item additions.

### UI
For now, I can imagine some simple expansions players would love to spend Coins to obtain. An expanded Favorite Beatmaps section, Bigger Friend Limit, or special presets that completely change how their profile pages look (Think Steam, Discord). The point is, any improvements to the UI (which I'll make another post on) will naturally tie into improvements that can be turned into rewards.

With osu!Lazer's continually expanding UI, we can expect more empty space players will want to decorate. For now, I'll need to reference other games to give a shared visual for this to make sense.

![7cgBwgt](https://github.com/user-attachments/assets/9d72f150-1b49-4f60-a40d-8c0e78f6c1ab)

There are a few key elements in here.
Not only is each section of the profile clearly defined (instead of being a long, borderless scroll), the focus is clear for each element.
There's a profile picture with a customizable border around it, on top of a customizable banner draping underneath. Achievements such as Rank, Honor (would probably translate into kudosu or special contributor badges), and Mastery (Levels or Medals).
All of this with a massive, slightly faded Backdrop Art that doesn't feel encumbered or weighed down by excess details due to a clear view of the center and thirds.

If osu profile pages get redesigned, there is plenty of room for new cosmetics and old achievements to all be showcased. 

For those that want to flex, Cosmetics such as Icons, Emotes (used in Chat), and Titles scaling in quality and effect corresponding to a player's present stats would be desirable.

### At the end of the day
The only thing at osu!'s core is its community-driven nature. So much is pushed out through essentially volunteer work. This has been a blessing to osu!, but also might be one of the sources of fear around "changing" to the game. I've tried to assess how this would impact the community:

The negatives I see are a serious lack of interest by a majority of the playerbase with any of the new features, similar to lazer in its initial stage and the Spotlights events. 

The positives, I feel, are more avenues to properly appreciate the Artists of osu!, reward Player's for their time, and give Developers another method to incentivize content. 

We need better ways to affirm unique player achievements. Rank always leads to sharp, direct comparisons. Medals go often overlooked. Skill is too nebulous to immediately grasp. Demand for cosmetics is there; every other game of a similar size proves it. Hell, even Discord managed did this. Can't we do this for osu?

**Comment by BlockHead007** [comment]
Sounds like arcadia, "Now with online features"
Also, somebody still remembers that osu!coins April fools' joke from 2015
